src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -146,36 +171,6 @@ def evaluate_workflows_action_filters(
     return filter_recently_fired_workflow_actions(filtered_action_groups, event_data)
 
 
-def log_fired_workflows(
-    log_name: str, actions: list[Action], event_data: WorkflowEventData
-) -> None:
-    # go from actions to workflows
-    action_ids = {action.id for action in actions}
-    action_conditions = DataConditionGroup.objects.filter(
-        dataconditiongroupaction__action_id__in=action_ids
-    ).values_list("id", flat=True)
-    workflows_to_fire = Workflow.objects.filter(
-        workflowdataconditiongroup__condition_group_id__in=action_conditions
-    )
-    workflow_to_rule = dict(
-        AlertRuleWorkflow.objects.filter(workflow__in=workflows_to_fire).values_list(
-            "workflow_id", "rule_id"
-        )
-    )
-
-    for workflow in workflows_to_fire:
-        logger.info(
-            log_name,
-            extra={
-                "workflow_id": workflow.id,
-                "rule_id": workflow_to_rule.get(workflow.id),
-                "payload": asdict(event_data),
-                "group_id": event_data.event.group_id,
-                "event_id": event_data.event.event_id,
-            },
-        )
```

Comment:
This log ended up feeling like it was combining concerns -- it has info for only issue rules + workflow engine. Instead, these logs have been updated in places to use only workflow engine, we can have logs outside of workflow engine (like what metric alerts is doing) to log the info rule data.